.go-version

```diff
@@ -1 +1 @@
-GOVERS = go1\.7.*
+GOVERS = go1\.7\.1
```

Comment:
This regexp should accept future versions, e.g. `go\1.7\.[1-9]`. Otherwise next time there won't be a chance for people to upgrade in advance.